dlt/pipeline/pipeline.py

```diff
@@ -724,7 +734,7 @@ def sync_destination(
 
             # write the state back
             self._props_to_state(state)
-            bump_version_if_modified(state)
+            bump_pipeline_state_version_if_modified(state)
```

Comment:
heh we are close to  implement a this as a class :)

Reply:
yeah, i think so too..